bin/configs/spring-boot-oas3.yaml

```diff
@@ -8,3 +8,4 @@ additionalProperties:
   artifactId: springboot
   snapshotVersion: "true"
   hideGenerationTimestamp: "true"
+  requestMappingMode: iface
```

Comment:
I used the shortcut here because **interface** is a reserved word in Java. I hope that is okay.

Reply:
minor suggestion: what about using `api_interface` instead of `iface` ?

people reading the config (e.g. overtaking someone else project) will instantly figure out what it's.